### PR TITLE
Explicitly import array from numpy in scatterplot tool.

### LINKED ISSUE #282: scatterplot crashes
python /usr/local/galaxy/shed_tools/toolshed.g2.bx.psu.edu/repos/devteam/scatterplot/c12b0759203b/scatterplot/scatterplot.py /data/2/galaxy_db/files/000/317/dataset_317644.dat /data/3/galaxy_db/files/000/317/dataset_317646.dat 2 1 "Scatterplot" "V1" "V2"

job stderr:
global name 'array' is not defined

### PR DESCRIPTION
**Pull request by nsoranzo**
Add numpy as requirement for rpy and scatterplot.

Should fix #282.

Not tested!

### CODE REVIEW

File: tools/scatterplot/scatterplot.xml [hunk 1]
```diff
@@ -1,71 +1,70 @@
...
+    <param name="col1" type="data_column" data_ref="input" numerical="True" label="Numerical column for x axis" />
+    <param name="col2" type="data_column" data_ref="input" numerical="True" label="Numerical column for y axis" />
+    <param name="title" type="text" value="Scatterplot" label="Plot title"/>
+    <param name="xlab" type="text" value="V1" label="Label for x axis"/>
+    <param name="ylab" type="text" value="V2" label="Label for y axis"/>
+  </inputs>
+  <outputs>
+    <data format="pdf" name="out_file1" />
+  </outputs>
+  <!-- TODO: uncomment the following test when we have tools.update_state() working for
```
Comment (bgruening) [inline tools/scatterplot/scatterplot.xml]: @nsoranzo did you tried to run this test? This warning looks strange.

**Comment by bgruening** [comment]
LGTM! :+1:

File: tools/scatterplot/scatterplot.xml (hunk 1 above)
Comment (nsoranzo) [inline tools/scatterplot/scatterplot.xml]: No, as mentioned in the PR description, I've not done testing, I was relying on you for that :)

**Comment by martenson** [comment]
thanks @nsoranzo

**Pull request merged**